### PR TITLE
[EN-46573] upgrade lodash and babel

### PR DESCRIPTION
**Pull request by vickingan**

### CODE REVIEW

File: es6-lib/config/index.js [hunk 1]
```diff
@@ -9,19 +9,27 @@
...
+import prodEnv from './prod.js';
+import testEnv from './test.js';
 
 var env = process.env.GEO_IMPORT_ENV;
 if (!env) throw new Error('No GEO_IMPORT_ENV environment set! Please specify one.');
 
-var baseConfig = require('./config');
-
-const conf = merge(baseConfig, require(`./${env}`));
+let conf;
```
Comment (vickingan) [inline es6-lib/config/index.js:22]: Big change here--no longer able to dynamically import.

File: bin/start.sh [hunk 2]
```diff
@@ -12,4 +12,4 @@ else
   n $node_version
 fi
 
-GEO_IMPORT_ENV=dev node lib/index.js
+GEO_IMPORT_ENV=dev ZOOKEEPER_ENSEMBLE="{}" node lib/index.js
```
Comment (vickingan) [inline bin/start.sh:15]: Must pass in `ZOOKEEPER_ENSEMBLE={}` now in order for `import prod.sh` to not break.

File: es6-test/unit/geojson.js [hunk 3]
```diff
@@ -1,25 +1,25 @@
...
+import { ArityChecker } from '../util.js';
+
 var expect = chai.expect;
 
 describe('geojson decoder', function() {
-
   it('will emit an error for malformed json', function(onDone) {
-    var count = 0;
+    const geoJsonInstance = new GeoJSON();
+    geoJsonInstance.prependListener('error', (error) => {});
```
Comment (vickingan) [inline es6-test/unit/geojson.js:13]: Changed test here--upgraded mocha so when unit test emits an error, the error is thrown and tests can't continue.

File: package.json [hunk 4]
```diff
@@ -2,9 +2,12 @@
   "name": "geo-import",
   "version": "0.1.89-SNAPSHOT",
   "private": true,
+  "type": "module",
   "dependencies": {
+    "@babel/cli": "7.14.5",
```
Comment (vickingan) [inline package.json:7]: Old lodash was tied to old babel, so needed to upgrade babel.

**Review by langlk — APPROVED** [review]

**Pull request merged**